spec/system/view_data_in_s3_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "View status of data in S3" do
+  before { sign_in user }
+
+  describe "when a dataset has a DOI and its data is in S3" do
+    let(:user) { FactoryBot.create :user }
+    let(:dataset) { FactoryBot.create :shakespeare_and_company_dataset }
+
+    it "shows data from S3", js: true do
+      visit dataset_path(dataset)
+      expect(page).to have_content dataset.title
+      # expect(page).to have_content "SCoData_combined_v1_2020-07_README.txt"
```

Comment:
This is not a criticism, but are these still intended to be comments? I just want to be certain that these weren't left accidentally.

Reply:
I should have said... I didn't actually get this working yet. I put these in to mark what I want to do, but they won't actually pass yet b/c I haven't hooked the S3 service up to the controller.